Makefile

```diff
@@ -0,0 +1,17 @@
+all: lint build
+
+.PHONY: build
+build:
+	build/build.sh
+
+.PHONY: lint
+lint:
```

Comment:
Any reason for calling it `lint`? We've a [golint](https://github.com/golang/lint) which is different from gofmt.

Reply:
`lint` is actually a very generic term. You can even lint for code complexity. Take a look at this [list](https://github.com/golangci/awesome-go-linters). I want to keep it generic in case we want to add more lint checks here.

Reply:
nit: I don't have a strong opinion. So you feel fmt is linting as well? Not sure if that list is coming from official golang docs?

Reply:
The term linting is generic and predates go. And yes linting includes formatting as well.

Reply:
https://en.wikipedia.org/wiki/Lint_(software)